core/arch/arm/arm.mk

```diff
@@ -115,7 +115,7 @@ arm32-platform-aflags-no-hard-float ?=
 
 arm64-platform-cflags-no-hard-float ?= -mgeneral-regs-only
 arm64-platform-cflags-hard-float ?=
-arm64-platform-cflags-generic ?= -mstrict-align
+arm64-platform-cflags-generic ?= -mstrict-align -mno-outline-atomics
```

Comment:
Please use `$(call cc-option,-mno-outline-atomics,)` for compatibility with older compilers.